src/pages/SideBar/SideBar.jsx

```diff
@@ -31,26 +35,36 @@ function SideBar(props) {
   };
 
   return (
-    <SideMenu fluid vertical tabular>
+    <SideMenu fluid vertical tabular >
       <Item
         icon="home"
         name="home"
         active={optionMenu === 'home'}
+        themeisdark={appContext.themeIsDark.toString()}
         onClick={() => changeMenu('home')}
       />
       <Item
         icon="heart outline"
         name="favorites"
         active={optionMenu === 'favorites'}
+        themeisdark={appContext.themeIsDark.toString()}
         onClick={() => changeMenu('favorites')}
       />
       <Item
         icon="clock outline"
         name="watchLater"
         active={optionMenu === 'watchLater'}
+        themeisdark={appContext.themeIsDark.toString()}
         onClick={() => changeMenu('watchLater')}
       />
-      <Menu.Item name="logOut" icon="log out" onClick={() => logout()} />
+      <Item 
+      themeisdark={appContext.themeIsDark.toString()} 
+      name="logOut" 
+      icon="log out" 
+      onClick={() => {
+        logout();
+      }} 
```

Comment:
`logout` can be passed as is because it is a function

```suggestion
      onClick={logout} 
```